bx_py_utils/test_utils/snapshot.py

```diff
@@ -1,7 +1,10 @@
 """
     Assert complex output via auto updated snapshot files with nice diff error messages.
+
+    To update all snapshot files, run your tests with RAISE_SNAPSHOT_ERRORS=0 in environment.
```

Comment:
I think this feature is so useful that we even should consider adding it to the `README.md`. Anyway, thanks for implementing!

Reply:
done.